internal/spentcalories/spentcalories.go

```diff
@@ -14,25 +19,154 @@ const (
 )
 
 func parseTraining(data string) (int, string, time.Duration, error) {
-	// TODO: реализовать функцию
+
+	parts := strings.Split(data, ",")
+
+	if len(parts) != 3 {
+		return 0, "", 0, errors.New("Ошибка: неверный формат данных")
+	}
+
+	stepsStr := strings.TrimSpace(parts[0])
+	steps, err := strconv.Atoi(stepsStr)
+	if err != nil {
+		return 0, "", 0, errors.New("Ошибка: неверно введены шаги")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
3. Ошибку лучше начинать с маленькой буквы
```suggestion
		return 0, "", 0, fmt.Errorf("conversion error: %w", err)
```